fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java

```diff
@@ -634,6 +634,10 @@ public static boolean checkTableAction(
         try {
             PrivilegeCollection collection = manager.mergePrivilegeCollection(context);
             return manager.checkTableAction(collection, db, table, action);
+        } catch (PrivObjNotFoundException e) {
```

Comment:
why not catch PrivObjNotFoundException checkAction and return true in checkAction?

Reply:
because how to handle the exception should leave to the caller, for some situation, we shouldn't catch PrivObjNotFoundException. And for some other method like checkAnyAction..., we can't catch the exception here.